guides/README.md

```diff
@@ -6,7 +6,7 @@ description: Angle Protocol Guides
 
 This section contains all guides to interact with the Angle Protocol.
 
-You will notably find some general user guides to onboard you to DeFi and to Angle first stablecoin (agEUR):
+Here are some preliminary guides to onboard you to DeFi and to agEUR, Angle's first stablecoin:
```

Comment:
Phrasing sounds weird: This section contains, and then Here are some preliminary guides.

We could say: You will notably find some preliminary guides to get onboarded to DeFi and to agEUR, Angle's first stablecoin